kubert/src/server.rs

```diff
@@ -46,27 +58,17 @@ pub struct SpawnedServer {
     local_addr: SocketAddr,
     task: tokio::task::JoinHandle<()>,
 }
-
-/// The path to the server's TLS private key
-#[derive(Clone, Debug)]
-#[cfg_attr(docsrs, doc(cfg(feature = "server")))]
-pub struct TlsKeyPath(PathBuf);
-
-/// The path to the server's TLS certificate bundle
-#[derive(Clone, Debug)]
-#[cfg_attr(docsrs, doc(cfg(feature = "server")))]
-pub struct TlsCertPath(PathBuf);
-
 /// Describes an error that occurred while initializing a server
 #[derive(Debug, Error)]
 #[cfg_attr(docsrs, doc(cfg(feature = "server")))]
+#[non_exhaustive]
 pub enum Error {
     /// No TLS key path was configured
-    #[error("--server-tls-key must be set")]
+    #[error("--server-tls-key must be set if TLS is enabled")]
```

Comment:
This is a user-facing error message. "if it's enabled" won't make sense to a user of the application...